src/pacmod3_ros_msg_handler.cpp

```diff
@@ -27,704 +30,738 @@ void LockedData::setData(std::vector<unsigned char> new_data)
   _data = new_data;
 }
 
-void Pacmod3TxRosMsgHandler::fillAndPublish(const int64_t& can_id,
-                                            std::string frame_id,
-                                            ros::Publisher& pub,
-                                            std::shared_ptr<Pacmod3TxMsg>& parser_class)
+void Pacmod3TxRosMsgHandler::fillAndPublish(
+const int64_t& can_id,
```

Comment:
This probably ought to be indented one
```suggestion
    const int64_t& can_id,
```